scheduler/reconcile_test.go

```diff
@@ -1763,11 +1767,12 @@ func TestReconciler_RescheduleNow_Service(t *testing.T) {
 		deploymentUpdates: nil,
 		place:             2,
 		inplace:           0,
-		stop:              0,
+		stop:              1,
```

Comment:
But some tests like this one has other things in play: here only one is rescheduled while the other placement is a replacement that's not considered a reschedule.  It's not quite so obvious by looking at the test besides the comment above `assertResults` line